client/src/components/PollsForReview/ImageOne.js

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { useContext } from 'react';
+import axios from 'axios';
+
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
+
+import UserContext from '../../context/UserContext';
+
+export default function ImageOne ({ userId, poll }) {
+    const { reload, setReload } = useContext(UserContext);
+    const { imageOneUrl, imageOneVotes, votedForImageOne, votedForImageTwo } = poll;
+    const pollId = poll._id;
+
+    const handleVote = async () => {
+        if (votedForImageTwo && votedForImageTwo.includes(userId)) {
+            return
+        } else if (votedForImageOne && votedForImageOne.includes(userId)) {
+            await removeVote();
+        } else await addVote();
+
+        setReload(!reload);
+    };
+
+    const addVote = async () => {
+        let token = localStorage.getItem('auth-token');
+        
+        try {
+            await axios.put('http://localhost:5000/polls/vote-one-add', 
```

Comment:
we can try to consolidate these repeated code (the only diff is image one or image two). for example this route, we can just introduce a `/vote-add` route, and pass in if it's `one` or `two` to the backend

Reply:
refactored.